src/buckets/components/BucketOverlayForm.tsx

```diff
@@ -1,13 +1,11 @@
 // Libraries
 import React, {PureComponent, ChangeEvent, FormEvent} from 'react'
-import moment from 'moment'
 
 // Components
 import {Form, Input, Button, Grid} from '@influxdata/clockface'
 import Retention from 'src/buckets/components/Retention'
 
 // Constants
-import {MIN_RETENTION_SECONDS} from 'src/buckets/constants'
```

Comment:
also, you can delete this from `src/buckets/constants/index.ts` - i just did a quick global search and it's only used here.

Reply:
Will do yes!!

Reply:
✅